logging.go

```diff
@@ -120,23 +103,16 @@ func requestScopeLogger(
 	buildVersion string,
 ) *Logger {
 	ctxLogger := logger.WithFields(logrus.Fields{
-		"app":            appName,
+		"application":    appName,
 		"env":            envName,
 		"build_version":  buildVersion,
 		"scope":          "request",
 		"correlation_id": correlationID,
-		"url":            r.RequestURI,
-		"route":          route,
+		"url":            r.URL.String(),
+		"path":           route,
 		"remote_addr":    r.RemoteAddr,
 		"method":         r.Method,
 		"ip":             ip,
-		"headers": logrus.Fields{
-			"host":              r.Host,
-			"user-agent":        r.UserAgent(),
-			"referer":           r.Referer(),
```

Comment:
I think these might be useful to have (host and referrer)

Reply:
these are just removed from every logging statement and placed specifically in the request log.

Reply:
ok